music21/expressions.py

```diff
@@ -325,6 +325,119 @@ def nextMark(self):
         return RehearsalMark(self.nextContent(), numbering=self.numbering)
 
 
+# ------------------------------------------------------------------------------
+class PedalType(common.StrEnum):
+    Sustain = 'sustain'
+    Sostenuto = 'sostenuto'
+    Soft = 'soft'
+    Silent = 'silent'
+
+class PedalForm(common.StrEnum):
+    Line = 'line'
+    Symbol = 'symbol'
+    SymbolAlt = 'symbolalt'
+    SymbolLine = 'symbolline'
+
+class PedalMark(spanner.Spanner):
+    '''
+        A pedal mark spanner contains a pedaled set of notes.
```

Comment:
Make sure to get the comments from the #1760 PR -- Music21 docs don't indent another 4 spaces.

Reply:
Done.